src/include/duckdb/execution/index/art/leaf.hpp

```diff
@@ -16,7 +16,7 @@ namespace duckdb {
 class Leaf : public Node {
 public:
 	Leaf(Key &value, unsigned depth, row_t row_id);
-
+	Leaf(Key &value, unsigned depth, unique_ptr<row_t[]> row_ids_p, idx_t num_elements_p);
```

Comment:
I know it was already there, but is there any reason why this is `unsigned depth` and not `uint32_t depth`?

Reply:
It's a relic from an earlier age. When I did it in '19 we didn't have defined code standards yet(And definitely no storage, where I think ```unsigned``` can be more tricky)
 ;-)

Reply:
Funny that this makes it feel like was in the long long ago

Reply:
Hah, nostalgia kicking in over old code. Love it when that happens

Reply:
I can go change this in this PR, but @pdet and me also found other relicts while going over the code together. Maybe a separate PR to address these would be better? (E.g. `GetChildPos` taking `uint8_t` instead of `idx_t` as a parameter I think).